docs/en/ingest-management/release-notes/release-notes-8.2.asciidoc

```diff
@@ -0,0 +1,248 @@
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/beats/issues/fleet-server/
+:fleet-server-pull: https://github.com/elastic/beats/pull/fleet-server/
+
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.2.0>>
+
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.2.0 relnotes
+
+[[release-notes-8.2.0]]
+== {fleet} and {agent} 8.2.0
+
+Review important information about the {fleet} and {agent} 8.2.0 release.
+
+[discrete]
+[[breaking-changes-8.2.0]]
+=== Breaking changes
+
+There are no breaking changes for {fleet} or {agent} in this release.
+
+[discrete]
+[[known-issues-8.2.0]]
+=== Known issues
+
+[[known-issue-issue-kibana-130758]]
+.During package upgrade, legacy component templates are not removed
+[%collapsible]
+====
+
+*Details* 
+
+In 8.2.0, when you upgrade an existing package, legacy `@settings` and
+`@mappings` component templates are not removed. These templates are deprecated
+in favor of the new `@package` component template. 
+
+*Impact* +
+
+The legacy templates do not cause issues, but you might want to deleted them
+because they are no longer used.
+
+To find and delete legacy component templates:
+
+. In {kib}, go to *Stack Management > Index Management* and click
+*Component Templates*.
+
+. Click `Not in use` to filter out component templates that are still in use.
+
+. If the list contains component templates that end in `@mappings` or
+`@settings`, use the Actions menu to delete them.
+
+====
+
+//REVIEWERS: Are there any //deprecations we need to document here? It sounds
+//like some component templates were deprecated, but not sure when.
+
+//keeping the template until I know whether we need it.
+
+//[discrete]
+//[[deprecations-8.2.0]]
+//=== Deprecations
+
+//The following functionality is deprecated in 8.2.0, and will be removed in
+//8.2.0. Deprecated functionality does not have an immediate impact on your
+//application, but we strongly recommend you make the necessary updates after you
+//upgrade to 8.2.0.
+
+//{fleet}::
+//* add info
+
+//{agent}::
+//* add info
+
+[discrete]
+[[enhancements-8.2.0]]
+=== Enhancements
+
+{fleet}::
+* beta[] Add a {ls} output type to {fleet} settings. {kib-issue}104987[#104987]
+* Merge the settings and mappings component templates into `@package`.
+{kib-pull}128498[#128498]
+* Redesign the agent flyout. {kib-pull}128381[#128381]
+* Add a link from agent details page to agent dashboard. {kib-pull}127882[#127882]
+* Update add agent instructions in {fleet}-managed mode for Kubernetes. {kib-pull}127703[#127703]
+* Add `time_series_metric` mapping for `metric_type` package field. {kib-pull}126322[#126322]
+* Add support for dimension field. {kib-pull}[#126257]
+* Refactor auto upgrade package policies logic. {kib-pull}125909[#125909]
```

Comment:
Since this is just a refactor effort, I think we can remove this from enhancements. I'll update the label on that PR